.vscode/settings.json

```diff
@@ -15,5 +15,11 @@
   },
   "[html]": {
     "editor.defaultFormatter": "esbenp.prettier-vscode"
-  }
+  },
+  "editor.codeActionsOnSave": {
```

Comment:
Hmm, this was a no go a couple of years back because it turns out most developers dislike this, and I don't remember much but it was functioning imperfectly back then. I will check it out and remove it later if it still is problematic